client/src/featureform/serving.py

```diff
@@ -433,18 +433,18 @@ def add_feature_dfs_to_list(self, feature_variant_list, entity_id):
         for feature_variant in feature_variant_list:
             f_name = feature_variant[0]
             f_variant = feature_variant[1]
-            f_category = self.db.get_feature_variant_category(f_name, f_variant)
+            f_mode = self.db.get_feature_variant_mode(f_name, f_variant)
 
-            if f_category == "ON_DEMAND_CLIENT":
+            if f_mode == "CLIENT_COMPUTED":
```

Comment:
make all of these consts, if something is a const and you're writing a string, one day someones gonna typo